tools/variant_simulator/simulate_variation

```diff
@@ -49,6 +49,7 @@ GetOptions( "species=s" => \$species,
     "codingOnly" => \$codingOnly,
     "intronsOnly" => \$intronOnly,
     "assembly=s" => \$assembly,
+    "refseq"     => \$refSeq_genes,
```

Comment:
For consistency, could we use the same case for `refseq` and `$refSeq_genes` ?
Usually `refseq` is written in Ensembl in lower case.

Reply:
Good point! I've updated this in the next commit.